entrypoint/go.mod

```diff
@@ -4,9 +4,6 @@ go 1.14
 
 require (
 	github.com/Icinga/go-libs v0.0.0-20200629140135-57268fcc7724
-	github.com/Icinga/icingadb v1.0.0-rc1
-	github.com/go-ini/ini v1.62.0
-	github.com/go-sql-driver/mysql v1.6.0
-	github.com/sirupsen/logrus v1.8.1
-	gopkg.in/ini.v1 v1.57.0 // indirect
+	github.com/icinga/icingadb v1.0.0-rc1.0.20210528122710-9fdcc839798a
```

Comment:
I don't like pinning any version here. This should better use `replace` to use the checkout of the icingadb repo that is used for the build.

Reply:
Once we have a stable version, @dependabot will take care of this.